examples/task_spesific_configuration.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+#
+# To make this run, you probably want to edit /etc/luigi/client.cfg and add something like:
+# DO NOT FORGET to set keep-alive as True.
```

Comment:
May I suggest that you include a command line on how to run this? Here's some inspiration: https://github.com/spotify/luigi/blob/5bd9b436990d48acc8def675382c29f8fb0e729e/examples/execution_summary_example.py#L21